Xamarin.Essentials/Launcher/Launcher.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Launcher
+    {
+        public static Task<bool> CanOpenAsync(string uri) => PlatformCanOpenAsync(new Uri(uri));
```

Comment:
should we do some validation of the string here? Usually we check for null or whitespace, throw an exception and then call the platform code.

Reply:
Done